src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_drm.c

```diff
@@ -967,11 +973,12 @@ static int xocl_cleanup_drm_memory_manager(struct xocl_mm *xocl_mm)
 	return 0;
 }
 
-static int xocl_init_drm_mm(struct xocl_drm *drm_p, struct xocl_mm *xocl_mm,
-		uint64_t mm_start_addr, uint64_t mm_end_addr, uint32_t m_count)
+static int xocl_init_drm_mm(struct xocl_drm *drm_p, struct xocl_mm *xocl_mm)
 {
 	int err = 0;
 	int i = 0;
+	uint64_t mm_end_addr = 0xffffFFFFffffFFFF;
```

Comment:
In Linux, you can use macro "[U64_MAX](https://elixir.bootlin.com/linux/v4.5/C/ident/U64_MAX)"